docs/src/test/java/docs/http/javadsl/HttpServerActorInteractionExample.java

```diff
@@ -105,13 +106,20 @@ private Route createRoute() {
 
   //#actor-interaction
   static class Auction extends UntypedActor {
+    List<Bid> bids = new ArrayList<>();
+
     static Props props() {
       return Props.create(Auction.class);
     }
 
     @Override
     public void onReceive(Object message) throws Throwable {
```

Comment:
Maybe you could use the lambda way as it's cleaner and it saves you to write castings and `instanceof`. Here you have the documentation just in case http://doc.akka.io/docs/akka/current/java/lambda-actors.html#Defining_an_Actor_class